week-1/interpret/percentage-change.js

```diff
@@ -12,11 +12,19 @@ console.log(`The percentage change is ${percentageChange}`);
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
+// function calls and lines where they are called are:
+// Number(carPrice.replaceAll()) and similarly in Line 5. Number() is a function while .replaceAll() is a method.
+// The second function is console.log() called in Line 10
 
 // b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
+// The code runs without error and produce a correct answer expected.
```

Comment:
You're right - I think this code is supposed to have a bug in it, but it looks good to me too.